core/src/main/java/org/elasticsearch/common/util/BigArrays.java

```diff
@@ -91,7 +90,7 @@ public void close() {
 
     private abstract static class AbstractArrayWrapper extends AbstractArray implements BigArray {
 
-        protected static final long SHALLOW_SIZE = RamUsageEstimator.shallowSizeOfInstance(ByteArrayWrapper.class);
+        static final long SHALLOW_SIZE = RamUsageEstimator.shallowSizeOfInstance(ByteArrayWrapper.class);
```

Comment:
@dakrone it needs to be package-private because the other array wrapper classes (e.g. `ByteArrayWrapper` access it.  Making it `private` throws compilation errors.